### PR TITLE
Adds new mipsolver_uses_internal_buffers parameter.

### PR DESCRIPTION
**Pull request by marcovc**
Since the MIP solver now also supports it.

### CODE REVIEW

File: crates/shared/src/arguments.rs [hunk 1]
```diff
@@ -135,6 +135,10 @@ pub struct Arguments {
     /// If quasimodo should use internal buffers to improve solution quality.
     #[structopt(long, env)]
     pub quasimodo_uses_internal_buffers: bool,
+
+    /// If mipsolver should use internal buffers to improve solution quality.
+    #[structopt(long, env)]
+    pub mipsolver_uses_internal_buffers: bool,
```
Comment (nlordell) [inline crates/shared/src/arguments.rs:141]: uber-naming-nit:
For consistency, can we name them either:
```
quasimodo_solver_uses_internal_buffers
mip_solver_uses_internal_buffers
```

Or (my personal preference):
```
quasimodo_uses_internal_buffers
mip_uses_internal_buffers
```

**Review by nlordell — APPROVED** [review]
LGTM, tiny naming nit, but I don't fee strongly about it.

File: crates/shared/src/arguments.rs (hunk 1 above)
Comment (marcovc) [reply crates/shared/src/arguments.rs:141]: Thanks, makes sense!

**Comment by nlordell** [comment]
@marcovc For such a simple change, feel free to merge with 1 approval.

**Pull request merged**